internal/prompt/prompt.go

```diff
@@ -75,7 +79,7 @@ func (p *Prompt) InputAndValidate(title, message string, defaultResponse *string
 			logging.Debug("Selecting default choice %s for Input prompt %s in non-interactive mode", *defaultResponse, title)
 			return *defaultResponse, nil
 		}
-		return "", locale.NewInputError("err_non_interactive_prompt", message)
+		return "", &NonInteractiveError{locale.NewInputError("err_non_interactive_prompt", message)}
```

Comment:
`err_non_interactive_prompt` does not appear to exist.

Reply:
It already exists in the locale file and has been used in existing non-interactive prompt error messages.